### PR TITLE
Change pronounced prop to intent in FormFieldLabel

### PR DESCRIPTION
**Pull request by libertymayc**
Discussed in dev call 18/7

### CODE REVIEW

File: .changeset/fresh-planets-talk.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Change `pronounced` prop to `intent` enum. Use "strong" for pronounced label styling
```
Comment (libertymayc) [inline .changeset/fresh-planets-talk.md:5]:
```suggestion
Change `pronounced` prop to `intent` enum. Use "sentence" for pronounced label styling
```

**Comment by libertymayc** [comment]
discussed with Ben, decided 'label' and 'sentence' are suitable values for intent

**Review by Fercas123 — APPROVED** [review]

**Review by ZarkoPernar — APPROVED** [review]

File: .changeset/fresh-planets-talk.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Change `pronounced` prop to `intent` enum. Use "sentence" for pronounced label styling
```
Comment (alycrys) [inline .changeset/fresh-planets-talk.md:5]: this sounds a bit confusing

File: packages/core/stories/form-field/form-field.doc.stories.mdx [hunk 3]
```diff
@@ -70,15 +70,15 @@ Place your label to the left of the control if your application is limited in ve
 
 ## Form Field Label Question
 
-The Form Field Label can be styled as a question by setting the `pronounced` prop to `true`.
+The Form Field Label can be styled as a question by setting the `intent` prop to `sentence`.
```
Comment (alycrys) [inline packages/core/stories/form-field/form-field.doc.stories.mdx:73]:
```suggestion
The Form Field Label can be emphasized by setting the `intent` prop to `sentence`.
```

File: .changeset/fresh-planets-talk.md (hunk 2 above)
Comment (libertymayc) [inline .changeset/fresh-planets-talk.md:5]:
```suggestion
Removed `pronounced` prop to
Added `intent` prop. Use `intent` with "sentence" value to replace pronounced label styling
```

Comment (libertymayc) [reply .changeset/fresh-planets-talk.md]: updated, lmk

File: .changeset/fresh-planets-talk.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/core": patch
+---
+
+Removed `pronounced` prop to
```
Comment (alycrys) [inline .changeset/fresh-planets-talk.md:5]:
```suggestion
Removed `pronounced` prop.
```

File: .changeset/fresh-planets-talk.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/core": patch
+---
+
+Removed `pronounced` prop.
+Added `intent` prop. Use `intent` with "sentence" value to replace pronounced label styling
```
Comment (joshwooding) [inline .changeset/fresh-planets-talk.md:6]:
```suggestion
The `pronounced` prop has been replaced with the `intent` prop.

```diff
- <FormLabel pronounced />
+ <FormLabel intent="sentence" />
```

**Review by joshwooding — CHANGES_REQUESTED** [review]

**Review by joshwooding — APPROVED** [review]

**Review by ZarkoPernar — APPROVED** [review]

**Review by alycrys — APPROVED** [review]

**Pull request merged**